doc/releases/migration-guide-3.7.rst

```diff
@@ -74,6 +74,15 @@ Analog-to-Digital Converter (ADC)
 Bluetooth HCI
 =============
 
+Charger
+=======
+
+* Dropped ``constant-charge-current-max-microamp`` property in ``charger_max20335`` driver because
```

Comment:
@kartben Could you please check if we should mention this in the migration guide?